src/components/quote.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+
+const Quote = () => {
+  const apiUrl = 'https://api.api-ninjas.com/v1/quotes';
+  const category = 'life';
+  const [quoteText, setQuoteText] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const response = await fetch(`${apiUrl}?category=${category}`, { headers: { 'X-Api-Key': 'ol8rLaI2WXeFoWWBPY4sQA==yP7NsdMF8EyLCJWx' } });
+        if (!response.ok) {
+          throw new Error(`API request failed with status code ${response.status}.`);
+        }
```

Comment:
- [ ] Also I don't think this is necessary since the catch block is there to handle any error

Reply:
**Hi @VICTOR-CHUKWU ,**

- Thank you for your time and for reviewing my code. 
- I believe that catch block may handle any error that occurs, but the purpose of my code is to provide a specific error message that indicates the cause of the error. This can be useful to quickly identify and resolve the issue, rather than having to dig through logs or other sources of information to determine the cause of the error.